### ISSUE TITLE
Web Api Example

### ISSUE THREAD

**Issue by pdevito3** [FusionAuth/fusionauth-example-asp-netcore#2]
👋🏼 team, 

Any chance of you guys getting a stand alone web api example up? Something production ready like [this](https://fusionauth.io/learn/expert-advice/authentication/spa/oauth-authorization-code-grant-jwts-refresh-tokens-cookies/) is a very common use case an recommended by you guys so i would would think it would be a good candidate. 

Personally, I'd like something where my API can use cookies if I'm on the same top-level domain, but have the option to just use a token if I'm not so I can consume my API from clients as well. This is similar to how [Sanctum](https://laravel.com/docs/8.x/sanctum#how-it-works-spa-authentication) is set up, though this would need oauth as well.

**Comment by mooreds** [comment]
@pdevito3 Thanks for the comment! Are you looking for this example in .NET Core or PHP, or just want an example in any language? 

Just a note; I write many of the example apps and typically don't write production ready example applications, they are more for exploring and tweaking. They often omit tests, instrumentation, logging and other things you'd want if you were deploying to prod.

**Comment by pdevito3** [comment]
Sorry, on a .NET project, but should have clarified. .NET 5 is what I'm looking for. Will likely overlap with 3.1 as well since they're similar.

When I mentioned production ready, I was more getting at the auth piece in startup itself. It seems like most of the examples are simple jwt's with on or maybe two tokens. Fine to illustrate it in a basic option, but feels like I'm coming out of it starting mostly from scratch to actually implement the recommended option. I'm generally fine to add tests, logging, etc. myself, but would like to see a complete auth setup for me to then log and test against (if that makes sense).

**Comment by mooreds** [comment]
@pdevito3 makes sense. I'll put this on the list of example apps to write. Thanks for the suggestion! Also would suggest posting in the forum to see if anyone has code to share: https://fusionauth.io/community/forum/

**Comment by pdevito3** [comment]
Awesome, thanks so much! Will be sure to post there as well in the meantime. Have a great weekend!
